src/error.rs

```diff
@@ -170,6 +170,13 @@ impl StdError for Error {
     }
 }
 
+#[cfg(target_arch = "wasm32")]
+impl From<crate::error::Error> for wasm_bindgen::JsValue {
+    fn from(err: Error) -> wasm_bindgen::JsValue {
+        wasm_bindgen::JsValue::from_str(&format!("{}", err))
```

Comment:
Should we be creating a value that is somehow `instanceof Error`? Or is that not easy to do?

Reply:
Interesting question - I had been assuming that Result::Err(JsValue) was good enough. I'll read through this thread for more guidance. 

https://github.com/rustwasm/wasm-bindgen/issues/1017

Reply:
This updated commit should now let users choose whether they wish to use js_sys::Error or wasm_bindgen::JsValue for their Result error type. The downside is that now errors *always* generate stack traces because they always create a js_sys::Error. 

Do you have any thoughts on this trade off?

Reply:
If you'd like, I can also add a section to the example to demonstrate these error messages.